workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,20 +49,22 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
         {
-            BuildConfig();
+            Launch("Launch standalone client", "local worker launch UnityClient default", "spatial_*.log");
+        }
 
-            var command = Common.SpatialBinary;
-            var commandArgs = "local launch";
-            
-            if (Application.platform == RuntimePlatform.OSXEditor)
-            {
-                command = "osascript";
-                commandArgs = $"-e 'tell application \"Terminal\"\nactivate\ndo script \"cd {SpatialProjectRootDir} && {Common.SpatialBinary} local launch\"\nend tell'";
-            }
+        public static void LaunchLocalDeployment()
+        {
+            Launch("Launch SpatialOS locally", "local launch", "*unityclient.log");
+        }
 
-            var processInfo = new ProcessStartInfo(command, commandArgs)
+        public static void Launch(string commandName, string commandArgs, string logFileName)
```

Comment:
Instead of passing a string - what about defining an enum?

So the signature would look something like: 

`public static void Launch(string commandName, LaunchType launchType, string logFileName, param string[] additionalArgs)`

Then its invoked like: `Launch("Launch standalone client", LaunchType.CLIENT, "logFile", "UnityClient", "default")` ? 

You could go further and infer the `commandName` and `logFile` from the `enum` as well.